### DISCUSSION TITLE
Dongle troubleshooting, not working?

### DISCUSSION THREAD

**Discussion by Moon4354** [makerdiary/nrf52840-mdk-usb-dongle#98] [Q&A]
Got this yesterday, plugged it in while holding the button, PC recognised it and i installed this file onto it https://github.com/koush/vysor.io/wiki/Vysor-Dongle#esp32 ( the UF2 flash file). and it worked perfectly. 

Today when I loaded up my PC only 1 green light turns on (the power), and my PC, or any other PC no longer detects it, I've tried it on 3 laptops and 2 other computers with other ports. Only 1 green light comes on and none of the pcs detect that I've plugged a USB in. 

Is there something I can do to make it work again? 
I've tried double-clicking it when plugged in, but that didn't work. 

Do I need to download something on my PC for it to be detected?

**Comment by punkuotukas** [comment]
Same here.
Tried plugging in on: 
- mac mini Intel Monterey 12.7.4
- macbook pro M3 Sonoma 14.4
- Dell Vostro 5502 Arch linux

When running `nrfutil device list`
outputs: `Found 0 supported device(s)`
